README.md

```diff
@@ -1,5 +1,8 @@
 # Erlang DNS Server
 
+[![Build Status](https://github.com/dnsimple/erldns/actions/workflows/ci.yml/badge.svg)](https://github.com/dnsimple/erldns/actions/workflows/ci.yml)
+[![Module Version](https://img.shields.io/hexpm/v/erldns.svg)](https://hex.pm/packages/erldns)
+
 Serve DNS authoritative responses... with Erlang.
 
 [![CI](https://github.com/dnsimple/erldns/actions/workflows/ci.yml/badge.svg)](https://github.com/dnsimple/erldns/actions/workflows/ci.yml)
```

Comment:
This info is now duplicated by the new badges

<img width="526" alt="Screenshot 2025-02-03 at 16 34 07" src="https://github.com/user-attachments/assets/44318f41-04b2-4233-b733-94488b09c592" />